src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,178 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import duke.exception.DukeException;
+import duke.exception.EmptyListException;
+import duke.exception.OutOfBoundsException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+@SuppressWarnings("ALL")
+public class TaskList {
+    private ArrayList<Task> list;
```

Comment:
Would naming the `list` variable as `tasks` better convey the contents of the `ArrayList`?